examples/eady_turbulence.jl

```diff
@@ -57,11 +57,18 @@ grid = RegularCartesianGrid(size=(Nh, Nh, Nz), halo=(2, 2, 2),
 
 bc_parameters = (μ=μ, H=Lz)
 
-@inline τ₁₃_linear_drag(i, j, grid, clock, state, p) = @inbounds p.μ * p.H * state.velocities.u[i, j, 1]
-@inline τ₂₃_linear_drag(i, j, grid, clock, state, p) = @inbounds p.μ * p.H * state.velocities.v[i, j, 1]
+@inline τ₁₃_linear_drag(i, j, grid, clock, state, params) =
+    @inbounds params.μ * params.H * state.velocities.u[i, j, 1]
```

Comment:
Why not use the `(x, y, t, p)` form here? Seems like it would be perfect.

Reply:
Because we have to access the field "u"! We need the depends_on keyword for that

Reply:
Ah getting conversations mixed up haha, forgot that `depends_on` isn't in yet!